CONTRIBUTING.md

```diff
@@ -73,6 +73,27 @@ The subcategory object should follow this pattern
 > - Your entries should follow the above structure.
 
 --- 
+## Issues
+* Select an issue template from [issues](https://github.com/EddieHubCommunity/LinkFree/issues/new/choose) tab. Otherwise choose **Other** if it doesn't match what you're looking for.
+* When creating an issue make sure you fill up all the fields properly.
+* Make sure that you are NOT raising a **duplicate issue**.
+* If you want to work on the issue, please tick *I am willing to work on this issue* checkmark. Then you can start working on the issue.
+* If you aren't the owner of the issue, please comment that you're willing to work on the issue and wait for maintainers to assign you the issue. Also don't work on the issue if you're NOT assigned.
+* Work on only ONE issue at a time
+  
+**Closing the issue**
+
+* While closing the issue, please leave a reasonable comment that Why you're closing the issue
+* If the Pull Request associated with issue get's mereged and the issue still remains open, it's you're responsibility to close the issue
+
+## Commits
+* Please keep your commit message short and clear.
+* Use `type: subject` format for writing your commit messages. `type` could be one the followings:
+  * `feat`: use this if you're adding any new feature
+  * `fix`: use this if you're fixing anything in the codebase
+  * `chore`: use this when you're adding new links/resources AND when making any small changes
+  (ex. chore: added *resource_name* in *subcategory* *category* )
+
```

Comment:
[Proofreading] I think adding the link to conventional commits can be helpful if contributors are still stuck on how to write effective commits. 
```suggestion
  (ex. chore: added *resource_name* in *subcategory* *category* )
If you need more tips, check out [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
```